composer.lock

```diff
@@ -4,16 +4,15 @@
         "Read more about it at https://getcomposer.org/doc/01-basic-usage.md#installing-dependencies",
```

Comment:
I think we may have to composer update again to remove the references to the testing tools we don't need anymore.